service/controller/clusterapi/v29/resource/tcnp/create.go

```diff
@@ -263,7 +271,7 @@ func (r *Resource) newSecurityGroups(ctx context.Context, cl v1alpha1.Cluster, m
 		},
 		TenantCluster: template.ParamsMainSecurityGroupsTenantCluster{
 			VPC: template.ParamsMainSecurityGroupsTenantClusterVPC{
-				ID: "", // TODO
+				ID: cc.Status.TenantCluster.TCCP.VPC.ID,
```

Comment:
Ninjaing this in here after discussions with Marcel.

Reply:
Did the same change in my PR just now :D
All good.